chichiwang/chichiwang.dev
# Create `Logo` component: SVG of personal logo

## Description
Linked to Issue: #2
Replace the [boilerplate](https://github.com/chichiwang/tamsui) logo with personal site logo. To this end, the old `app/components/Logo` directory is removed and replaced by a new `app/components/Logo` that renders an SVG of the new logo.

In addition to this, Tamsui's `app/components/HeroBanner` is also removed. References are updated in `pages/Home`, `pages/ErrorPage`, `pages/NotFound` and `app/Layout/Nav`. These are not styled to be production-level changes, but rather placeholder implementations to ensure that the logo looks correct.

## Changes
* `app/components/HeroBanner` removed
* `app/components/Logo` removed and replaced with a new `app/components/Logo`
* Update references to `app/components/Logo` in:
  * `pages/Home/index.tsx`
  * `pages/ErrorPage/index.tsx`
  * `pages/NotFound/index.tsx`
  * `app/Layout/Nav.tsx`

## Steps to QA
* Pull down and switch to this branch
* Run the app and ensure no issues
* Verify in browser that the new Logo being rendered is Chi-chi Wang's logo, not Tamsui
